content/secrets/05-walkthrough-step-02/cdk.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+Next, lets look at the stacks.   The files in `lib` each represent a Cloudformation Stack containing the component parts of the application infrastructure.  
```

Comment:
Rather than have these in separate pages, let's follow the format in the microservices chapter. Here's the example: https://ecsworkshop.com/microservices/frontend/